atmel-rf-driver/NanostackRfPhyAtmel.h

```diff
@@ -52,14 +55,33 @@
 #if !defined(ATMEL_I2C_SCL)
 #define ATMEL_I2C_SCL    D15
 #endif
+#if !defined(ATMEL_SPI_TEST1)
+#define ATMEL_SPI_TEST1    D6
+#endif
+#if !defined(ATMEL_SPI_TEST2)
+#define ATMEL_SPI_TEST2    D3
+#endif
+#if !defined(ATMEL_SPI_TEST3)
+#define ATMEL_SPI_TEST3    D4
+#endif
+#if !defined(ATMEL_SPI_TEST4)
+#define ATMEL_SPI_TEST4    D2
+#endif
+#if !defined(ATMEL_SPI_TEST5)
+#define ATMEL_SPI_TEST5    D8
+#endif
 
 class RFBits;
 
 class NanostackRfPhyAtmel : public NanostackRfPhy {
 public:
     NanostackRfPhyAtmel(PinName spi_mosi, PinName spi_miso,
                         PinName spi_sclk, PinName spi_cs,  PinName spi_rst, PinName spi_slp, PinName spi_irq,
-                        PinName i2c_sda, PinName i2c_scl);
+                        PinName i2c_sda, PinName i2c_scl
```

Comment:
These new parameters should be moved to new function that is used only for testing purposes
The parameter names should be more descriptives example RX start TX done ....

Reply:
That's good idea. I'll create a separate task for this.